test/example-types/facet.js

```diff
@@ -434,5 +435,82 @@ describe('facet', () => {
         ],
       })
     })
+
+    describe('should always include checked values', () => {
+      let Data = [
+        { _id: 1, name: '1' },
+        { _id: 2, name: '2' },
+        { _id: 3, name: '3' },
+        { _id: 4, name: '4' },
+      ]
+      let mongoIdData = [
+        { _id: '5e9dbd76e991760021124966', name: 'Automation' },
+        { _id: '5cde2658dc766b0030c67dae', name: 'Fowlkes (MO)' },
+        { _id: '5d1ca49436e1d20038f8c84f', name: 'Customer Experience' },
+        { _id: '5ce30b403aa154002d01b9ed', name: 'Government Division' },
+      ]
+      let node = {
+        key: 'id',
+        field: '_id',
+        type: 'facet',
+        label: {
+          collection: null,
+          foreignField: '_id',
+          fields: ['name'],
+        },
+        mode: 'include',
+        optionsFilter: '',
+        size: 2,
+      }
+      it('when there are missed values', async () => {
+        ;(node.label.collection = Data), (node.values = [4])
```

Comment:
what's the weird syntax for?

Reply:
It was
 node.label.collection = Data 
node.values = [4]
it automatically formats to this. let me confirm is this autoformat by my local prettier or the DUTI

Reply:
that is duti format the code in my local is 
![image](https://user-images.githubusercontent.com/10645051/92289681-43c01180-eedf-11ea-9383-3f6b693d0f27.png)

Reply:
the comma caused the DUTI formating. After removed the comma, there no  weird syntax